osfclient/cli.py

```diff
@@ -200,6 +204,10 @@ def fetch(args):
     store = project.storage(storage)
     for file_ in store.files:
         if norm_remote_path(file_.path) == remote_path:
+            if local_path_exists and not args.force and args.update:
+                if file_.hashes['sha256'] == sha256_checksum(local_path):
+                    print("Local file %s already matches remote." % local_path)
+                    break
```

Comment:
Should this be `continue` instead of `break`? From reading the local code it seems we are iterating over all the files in a project and if one of them doesn't need updating we should skip it but continue with the rest. Instead of ending the `for` loop which is what `break` does. If we should use a `break` then I think we should have a comment explaining why, because I can already see future Tim coming back here and being confused again.